frontend/src/components/GoalForm/GoalRttapa.js

```diff
@@ -1,7 +1,8 @@
-import React, { useMemo } from 'react';
+import React, { useMemo, useRef } from 'react';
 import PropTypes from 'prop-types';
 import { FormGroup, Fieldset, Radio } from '@trussworks/react-uswds';
 import Req from '../Req';
+import Drawer from '../Drawer';
```

Comment:
I almost wonder if we want to move this up to site NAV level so we can reuse it for messages on other pages.

Reply:
Just pass in the content it displays as a child.

Reply:
right now it's at `src/components/Drawer.js`. I'm fine with moving it somewhere else - where were you thinking?